### ISSUE TITLE
Confusion about Forward Kinematics with Root Bone

### ISSUE THREAD

**Issue by jayden-ong** [dilevin/computer-graphics-kinematics#41]
In the README, it states: "As a base case, the root transformation can be defined to be the identity (no transformation)." 

If the rigid transformation for the root bone is the identity, does that mean the root bone never moves from its rest position?

**Comment by Zhecheng-Wang** [comment]
Yes. By rest position, we are referring to the relative rest position with respect to the root node.
Since we are defining the rest position with respect to the root node, obviously the root node with respect to the root node doesn't move.
